component/cargo/xs/src/cli.rs

```diff
@@ -26,6 +28,24 @@ where
             Some("--error-server-quit") => {
                 opts.err_server_quit = true;
             }
+            Some("--env-file") => {
+                let env_file_path = i.next();
+                let env_file_path =
+                    te!(env_file_path, "Missing argument to --env-file")
+                        .as_ref();
+                let env_file = std::fs::File::open(env_file_path)?;
+                match serde_yaml::from_reader::<_, EnvFile>(&env_file) {
+                    Ok(envs) => opts.envs = envs,
+                    Err(err) => println!("{:?}", err),
```

Comment:
Could not make use of serde_yaml::error::Error 😞

Reply:
Check this out: https://github.com/rx-ms/ct/pull/1